exonum/src/runtime/execution_context.rs

```diff
@@ -236,6 +255,11 @@ pub trait ExecutionContextUnstable {
     /// Invokes the interface method of the instance with the specified ID.
     ///
     /// See explanation about [`fallthrough_auth`](struct.ExecutionContext.html#child_context).
+    ///
+    /// # Panics
```

Comment:
Nit: I'd rather name this section "Return value", since the call does not panic *itself*. Also, I'd suggest to bubble errors up (e.g., via `?`) to avoid panics.